cmd/queue/main.go

```diff
@@ -319,8 +321,38 @@ func main() {
 			}
 		})
 
+		flush(logger)
 		if err := adminServer.Shutdown(context.Background()); err != nil {
 			logger.Errorw("Failed to shutdown admin-server", zap.Error(err))
 		}
 	}
 }
+
+func pushRequestLogHandler(currentHandler http.Handler) http.Handler {
+	templ := os.Getenv("SERVING_REQUEST_LOG_TEMPLATE")
+	if templ == "" {
+		return currentHandler
+	}
+
+	revInfo := &queue.RequestLogRevInfo{
+		Name:          os.Getenv("SERVING_REVISION"),
```

Comment:
Most all of this is already pulled from the environment in initEnv(). Why not add the additional environment variables in this struct above just reference the variables here?